plugins/inputs/opensearch_query/README.md

```diff
@@ -0,0 +1,258 @@
+# OpenSearch Query Input Plugin
+
+This [OpenSearch](https://opensearch.org/) plugin queries endpoints
+to derive metrics from data stored in an OpenSearch cluster.
+
+The following is supported:
+
+- return number of hits for a search query
+- calculate the `avg`/`max`/`min`/`sum` for a numeric field, filtered by a query,
+  aggregated per tag
+- `value_count` returns the number of documents for a particular field
+- `stats` (returns `sum`, `min`, `max`, `avg`, and `value_count` in one query)
+- extended_stats (`stats` plus stats such as sum of squares, variance, and standard
+  deviation)
+- `percentiles` returns the 1st, 5th, 25th, 50th, 75th, 95th, and 99th percentiles
+
+## OpenSearch Support
+
+This plugins is tested against OpenSearch 2.4.x.
```

Comment:
I tried finding it in previous conversations, but was this ever tested with 1.x?

Reply:
Yes.  It runs fine against 1.3.7.  I added the integration test suite for that container in 372c42557fc7964e6bcbdf87d1157880943f7538